contracts/contracts/solidity/utils/utils.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >0.6.0 <=0.8.0;
+
+/**
+ * @notice toUInt256 convert a bool to uint256.
+ */
+function toUInt256(bool x) pure returns (uint256 r) {
+    assembly {
+        r := x
+    }
+}
+
+/**
+ * @notice addressToUint convert an address to uint256.
+ */
+function addressToUint(address value) pure returns (uint256 convertedValue) {
```

Comment:
This function doesn't seem to be used anymore, and perhaps we inline the `toUInt256` as it doesn't make too much sense to keep an extra library for this three-liner fn.